src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/courtsentencing/CourtSentencingService.kt

```diff
@@ -432,11 +506,17 @@ fun LegacyCourtAppearance.toNomisCourtAppearance(
     courtEventType = "CRT",
     courtId = this.courtCode,
     outcomeReasonCode = this.nomisOutcomeCode,
-    nextEventDateTime = this.legacyData?.nextEventDateTime?.toString(),
+    nextEventDateTime = this.nextCourtAppearance?.let { next ->
+      next.appearanceTime?.let {
+        LocalDateTime.of(
+          next.appearanceDate,
+          LocalTime.parse(it),
```

Comment:
the dto from DPS has the date as a localDate and the optional time as a string 😢